python/ray/autoscaler/command_runner.py

```diff
@@ -717,8 +721,14 @@ def run_init(self, *, as_head, file_mounts):
 
             self.run("docker pull {}".format(image), run_env="host")
 
+        # Don't mount bootstrap files because of docker mount inode behavior.
```

Comment:
Can you more directly mention something along the lines of "When mounting, docker opens the underlying inode. Therefore if the file is switched, docker's inode is outdated."?

Reply:
Sounds good!